src/jsx/pages/Profiles/profile.js

```diff
@@ -0,0 +1,160 @@
+             activePag.current * sort,
+             (activePag.current + 1) * sort
+          )
+       ); */
+    };
+    return (
+       <div className="col-12">
+          <div className="card">
+             <div className="card-header">
+                <h4 className="card-title">Profile Datatable</h4>
+             </div>
+             <div className="card-body">
+                <Table responsive className="w-100">
+                   <div id="example_wrapper" className="dataTables_wrapper">
+                      <table id="example" className="display w-100 dataTable">
+                         <thead>
+                            <tr role="row">
+                               {profileTable.columns.map((d, i) => (
+                                  <th key={i}>{d}</th>
+                               ))}
+                            </tr>
+                         </thead>
+                         <tbody>
+                            {jobData.current.map((d, i) => (
+                               <tr key={i}>
+                                  {d.map((da, i) => (
+                                     <Fragment key={i}>
+                                        <td>
+                                           {i === 0 ? (
+                                              <img
+                                                 className="rounded-circle"
+                                                 width="35"
+                                                 src={da}
+                                                 alt=""
+                                              />
+                                           ) : (
+                                              <Fragment>
+                                                 {da}
+                                                 {i === 8 && (
+                                                    <div className="d-flex">
+                                                       <Link
+                                                          to="#"
+                                                          className="btn btn-primary shadow btn-xs sharp mr-1"
+                                                       >
+                                                          <i className="fa fa-pencil"></i>
+                                                       </Link>
+                                                       <Link
+                                                          to="#"
+                                                          className="btn btn-danger shadow btn-xs sharp"
+                                                       >
+                                                          <i className="fa fa-trash"></i>
+                                                       </Link>
+                                                    </div>
+                                                 )}
+                                              </Fragment>
```

Comment:
currently an image sits where a line of text should be. should we have profile images available? if not, what change should i make to this Fragment component to make this username sit in the profile tab correctly.